README.md

```diff
@@ -37,3 +38,56 @@ The Magento cron has a bad habit of getting stuck. **In fact, it's one of the mo
 The `send` method will be called on each model specified under the measurement group. In `send` you can get the data you need and send it to InfluxDB using `SomethingDigital_InfluxDb_Model_Api`. See the measurements in `app/code/community/SomethingDigital/InfluxDb/Model/Measurement` for reference.
 
 You can add additional groups and add additional measurements to groups as needed.
+
+### Sending Routing info in custom response header
+
+Routing information for a given request can be sent in a response header. A common use case for this would be to see response codes counts on a route-by-route basis. A new Apache CustomLog format can be used to save the response headers to the log files.
+
+#### Turning on the custom header
+
+**`local.xml`**
+
+```xml
+<config>
+    <sd_influxdb>
+        <route_response_headers>
+            <enabled>1</enabled>
+        </route_response_headers>
+    </sd_influxdb>
+</config>
+```
+
+In order to add a response headers to full page cache hits `SomethingDigital_InfluxDb_Model_RequestProcessor` also needs to be registered as a request processor and **must go last**.
```

Comment:
FWIW, it processes files in `app/etc/*.xml` in order, so using anything lexographically after `enterprise.xml` will make it go last.